ult)  # Should work without specifying log

```diff
@@ -0,0 +1,1202 @@
+Vignettes with name or keyword or title matching ‘plot’ using regular
```

Comment:
Hi, thanks for the update. I'm not sure what this file is but it has accidentally been included in the commit. Could you please have a look at that, and then I'll take a look on your update.